front/backfill_scripts/backfill_runs.ts

```diff
@@ -0,0 +1,75 @@
+import { App, Run } from "@app/lib/models";
```

Comment:
let's have this in `front/migrations` 

And let's use the same nomenclature as:
- https://github.com/dust-tt/dust/tree/main/core/src/stores/migrations
- https://github.com/dust-tt/dust/tree/main/front/mailing

`{date}_{name}.ts`